src/main/java/Decoder.java

```diff
@@ -0,0 +1,17 @@
+public class Decoder {
+    public static Command parseInput(String userInput) {
+        Command newCommand;
+        if (userInput.equals("bye")) {
+            newCommand = new ExitProgramCommand();
+        } else if (userInput.equals("list")) {
```

Comment:
Try to use CONSTANTS to improve readability.

Reply:
Noted. I would take this into consideration.